docs/user/apiv1.md

```diff
@@ -0,0 +1 @@
+Though the regular user will only need to interact with MasterMind's UI, once can perform all of MasterMind's operations via its API if needed. Assuming MasterMind is running locally, full specifications for the MasterMind API can be displayed by checking <http://localhost:3000/apidocs/index.html>
```

Comment:
we should publish it also somewhere online, so people can check it without deploying it.

Reply:
question is, on which resources hehe.

Reply:
@chicco785 I'll need to find somewhere to do so. The issue with using Swaggerhub or Apiary for instance is that I have a folder with multiple json files for my swagger specs, and those sites seem to only accept the yml formatted swagger specs. I can't just convert the json files with the official Swagger Editor either, because it only seems to accept one single large json file. I'll need to explore some options for this point.